src/data/faq.json

```diff
@@ -376,6 +379,217 @@
+                            {
+                                "title": "How long will the test remain visible in the app?",
+                                "anchor": "warn_without_tan_in_app_display",
+                                "active": false,
+                                "textblock": [
+                                    "Warning others with the new \"Manage Your Tests and Warn Others\" feature does not display a positive test result in the Corona-Warn-App at all.",
+                                    "For tests that are registered in the app as before, the same still applies: Tests always represent only a snapshot in the diagnosis of infectious diseases, they are therefore not valid indefinitely. After 72 hours, a registered test will therefore be grayed out as \"out of date\" in the Corona-Warn-App."
+                                ]
+                            },
+                            {
+                                "title": "After sharing the positive result, does my risk assessment stop? How long?",
+                                "anchor": "warn_without_tan_risk_assessment",
+                                "active": false,
+                                "textblock": [
+                                    "No, risk assessment and the contact tracing protocol are not stopped. Warning others with the new \"Manage Your Tests and Warn Others\" feature does not affect risk assessment.",
+                                    "If you are out in public despite testing positive, please be mindful of spacing and hygiene rules and, if possible, wear a mask to reduce a risk of infection to others."
+                                ]
+                            },
+                            {
+                                "title": "Does the risk calculation for an alert on my smartphone differ by test type from the person who tested positive?",
+                                "anchor": "warn_without_tan_risk_assessment_different",
+                                "active": false,
+                                "textblock": [
+                                    "No, the procedure for calculating the risk of infection has not been changed. Regardless of the type of test used to detect infection, the same parameters apply when calculating the risk for others."
+                                ]
+                            },
+                            {
+                                "title": "Does a warning based on the new feature disappear from the app by itself, or do I have to delete it independently?",
+                                "anchor": "warn_without_tan_warning",
+                                "active": false,
+                                "textblock": [
+                                    "No, this is not necessary. After a period of 10 days at the most, the warning in the app is automatically reset because the time window for the risk warnings is then also expired."

```

Comment:
Since there are two questions in the title, the answers have to be clear about which of the two questions are being answered.

So I suggest removing the first sentence and changing "is then" to "will have then".

 "After a period of 10 days at the most, the warning in the app is automatically reset because the time window for the risk warnings will have then also expired."

Also if the part "No, this is not necessary." is left in then you would need to explain whether it is even possible or not to delete a warning.

Reply:
We're going to delete this entry in the next commit, thanks for your feedback

Reply:
Deleted in https://github.com/corona-warn-app/cwa-website/pull/3322/commits/ff61d079ed080cd3b21a3a6996709f3b3f04ad26, thanks